crates/ibc/src/core/ics04_channel/msgs/chan_open_try.rs

```diff
@@ -20,7 +22,11 @@ pub const TYPE_URL: &str = "/ibc.core.channel.v1.MsgChannelOpenTry";
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct MsgChannelOpenTry {
     pub port_id_on_b: PortId,
-    pub chan_end_on_b: ChannelEnd,
+    pub ordering_on_b: Order,
+    pub version_on_b: Version,
+    pub connection_hops_on_b: Vec<ConnectionId>,
+    pub port_id_on_a: PortId,
+    pub chan_id_on_a: Option<ChannelId>,
     pub version_on_a: Version,
```

Comment:
I now think a better name for this would be `version_supported_on_a`, as it better represents what the version field means at this point in the handshake.